homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A scripting or programming language that allows you to implement complex features on web pages 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1.Git is a version control system that lets you manage and keep track of your source code history. 
+2.GitHub is a cloud-based hosting service that lets you manage Git repositories. 
+3.If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You create a personal copy of someone else's project, this allows you to later submit your contribution before affecting changes to the original project.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+it is copied onto your local computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you open a pull request, you're proposing your changes and requesting that someone review and pull in your contribution and merge them into their branch
```

Comment:
There can also be a case where you are the owner of the repository and you are asking your team members to review your code. In this case, you may be the person responsible for the merge not the person who reviews the code.